### PR TITLE
Ready for new ntuples

### PR DESCRIPTION
**Pull request by mverzett**
@bainbrid please check that everything makes sense to you. If so I'll submit the new ntuples right away

### CODE REVIEW

File: plugins/TrackerElectronsFeatures.cc [hunk 1]
```diff
@@ -210,6 +211,18 @@ TrackerElectronsFeatures::TrackerElectronsFeatures(const ParameterSet& cfg):
 //
 // member functions
 //
+vector<float> 
+TrackerElectronsFeatures::get_iso_rings(const GsfElectronRef& ele, const vector<reco::TrackRef> &tracks) {
+	vector<float> ret = {0., 0., 0., 0.};
+	// for(const auto& trk : tracks) {
+	//  	double dr = deltaR(*ele, *trk);
+	//  	size_t idx = std::floor(dr/0.1);
+	//  	if(idx >= ret.size())
```
Comment (bainbrid) [inline plugins/TrackerElectronsFeatures.cc]: @mverzett, should this not read ```if(idx <= ret.size())```?

File: plugins/TrackerElectronsFeatures.cc [hunk 2]
```diff
@@ -210,6 +211,18 @@ TrackerElectronsFeatures::TrackerElectronsFeatures(const ParameterSet& cfg):
 //
 // member functions
 //
+vector<float> 
+TrackerElectronsFeatures::get_iso_rings(const GsfElectronRef& ele, const vector<reco::TrackRef> &tracks) {
+	vector<float> ret = {0., 0., 0., 0.};
+	// for(const auto& trk : tracks) {
```
Comment (bainbrid) [inline plugins/TrackerElectronsFeatures.cc]: @mverzett, does this include the contribution from the electron itself? i.e., the ktf track from the electron is found in the ```tracks``` collection?

File: plugins/TrackerElectronsFeatures.cc (hunk 1 above)
Comment (mverzett) [reply plugins/TrackerElectronsFeatures.cc]: right...

File: plugins/TrackerElectronsFeatures.cc (hunk 2 above)
Comment (mverzett) [reply plugins/TrackerElectronsFeatures.cc]: yes, it's easier to remove it afterwards than at this stage

File: interface/ElectronNtuple.h [hunk 3]
```diff
@@ -238,6 +240,10 @@ class ElectronNtuple {
 	float ele_mvaIdV1_ = -2.;
 	float ele_mvaIdV2_ = -2.;
 	float ele_conv_vtx_fit_prob_ = -1.;
+	float ele_iso01_ = 0.;
+	float ele_iso02_ = 0.;
+	float ele_iso03_ = 0.;
+	float ele_iso04_ = 0.;
```
Comment (bainbrid) [inline interface/ElectronNtuple.h:246]: @mverzett, default these to a nonphysical -ve number? (I don't have a strong opinion)

File: interface/ElectronNtuple.h [hunk 4]
```diff
@@ -60,7 +61,8 @@ class ElectronNtuple {
 	void fill_gen(const reco::GenParticleRef genp);
 	void fill_gsf_trk(const reco::GsfTrackRef trk, const reco::BeamSpot &spot);
 	void fill_preid(const reco::PreId &preid, const reco::BeamSpot &spot, const int num_gsf);
-	void fill_ele(const reco::GsfElectronRef ele, float mvaid_v1=-2, float mvaid_v2=-2, float ele_conv_vtx_fit_prob = -1.);
+	void fill_ele(const reco::GsfElectronRef ele, float mvaid_v1=-2, float mvaid_v2=-2, 
+								float ele_conv_vtx_fit_prob = -1., const std::vector<float>& iso_rings={0., 0., 0., 0.});
```
Comment (bainbrid) [inline interface/ElectronNtuple.h:65]: @mverzett, default the iso_rings to a nonphysical -ve number? (I don't have a strong opinion)

File: plugins/TrackerElectronsFeatures.cc (hunk 2 above)
Comment (mverzett) [reply plugins/TrackerElectronsFeatures.cc]: actually I can try to remove it at this stage already

**Pull request merged**